src/devices/Ft232H/Ft232HDevice.cs

```diff
@@ -0,0 +1,857 @@
+                case "C3":
+                case "GPIOH3":
+                    return 11;
+                case "ACBUS4":
+                case "C4":
+                case "GPIOH4":
+                    return 12;
+                case "ACBUS5":
+                case "C5":
+                case "GPIOH5":
+                    return 13;
+                case "ACBUS6":
+                case "C6":
+                case "GPIOH6":
+                    return 14;
+                case "ACBUS7":
+                case "C7":
+                case "GPIOH7":
+                    return 15;
+                default:
+                    return -1;
+            }
+        }
+
+        /// <summary>
+        /// Instantiates a FT232H device object.
+        /// </summary>
+        /// <param name="flags">Indicates device state.</param>
+        /// <param name="type">Indicates the device type.</param>
+        /// <param name="id">The Vendor ID and Product ID of the device.</param>
+        /// <param name="locId">The physical location identifier of the device.</param>
+        /// <param name="serialNumber">The device serial number.</param>
+        /// <param name="description">The device description.</param>
+        public Ft232HDevice(FtFlag flags, FtDeviceType type, uint id, uint locId, string serialNumber, string description)
+        : base(flags, type, id, locId, serialNumber, description)
+        {
+        }
+
+        /// <summary>
+        /// Instantiates a FT232H device object.
+        /// </summary>
+        /// <param name="ftDevice">a FT Device</param>
+        public Ft232HDevice(FtDevice ftDevice)
+            : base(ftDevice.Flags, ftDevice.Type, ftDevice.Id, ftDevice.LocId, ftDevice.SerialNumber, ftDevice.Description)
+        {
+        }
+
+        /// <summary>
+        /// Creates I2C bus related to this device
+        /// </summary>
+        /// <returns>I2cBus instance</returns>
+        /// <remarks>You can create either an I2C, either an SPI device.</remarks>
+        public I2cBus CreateI2cBus() => new Ft232HI2cBus(this);
+
+        /// <summary>
+        /// Creates SPI device related to this device
+        /// </summary>
+        /// <param name="settings">The SPI settings</param>
+        /// <returns>a SPI device</returns>
+        /// <remarks>You can create either an I2C, either an SPI device.
+        /// You can create multiple SPI devices, the first one will be the one used for the clock frequency.
+        /// They all have to have different Chip Select. You can use any of the 3 to 15 pin for this function.</remarks>
+        public SpiDevice CreateSpiDevice(SpiConnectionSettings settings) => new Ft232HSpi(settings, this);
+
+        /// <summary>
+        /// Creates GPIO driver related to this device
+        /// </summary>
+        /// <returns>A GPIO Driver</returns>
+        public GpioDriver CreateGpioDriver() => new Ft232HGpio(this);
+
+        internal bool IsI2cMode { get; set; }
```

Comment:
I'd suggest make this public and abstract, non-settable in FtDevice, perhaps rename to `IsI2cAvailable` then `GetI2cBus` could throw if it's not available. Similar for SPI/GPIO

Reply:
The IsI2cAvailable, IsSpiAvailable, IsGpioAvailable is a good idea. I can add. The IsI2cMode is there to say that I2C is activated. So I can rename this one as IsI2cModeEnabled, can do the same for the 2 others.
As for this board, you can't have both I2c and Spi activated, I had to have this way to check.
What do you say about the properties to expose what's available and not?